client/components/EmployerReviews.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import SearchIcon from '@material-ui/icons/Search';
+
+const fakeReviews = [
+  {
+    id: 1,
+    review: 'Jobsite makes it so easy to reach a large, targeted candidate pool in a short amount of time',
+    name: 'Evans Ding, Software Engineer at Facebook',
```

Comment:
We might want to change the titles/companies for some of these to capture different industries!